### PR TITLE
Assigning maintainers against tests.

### PR DESCRIPTION
**Pull request by orsenthil**
### Define a Concept of Test Owners for Tests in DCOS/DCOS

https://jira.mesosphere.com/browse/DCOS-20355 

When a status check fails, we should make it easy for a developer to be self-sufficient, by helping the developer in identifying the purpose of the test, and provide information on the maintainer for the test.

Along those lines, we'd like to give [information about the various pull request statuses](https://github.com/dcos/dcos/pull/2320) and also give a `_maintainer_` information for the module. These could be useful when reviewing, assigning reviewers, or assisting with test failures.

####  Please Review

**Please review this PR** and share your feedback.

* I have assigned members from teams who, I thought, have worked on that piece of code.
* If a different maintainer is desired, please comment on this.
* Similar exercise will be done for the dcos-enterprise as well.

### CODE REVIEW

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

File: packages/dcos-integration-test/extra/test_mesos.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 import test_helpers
 from dcos_test_utils import marathon, recordio
 
+__maintainer__ = 'adam-mesos, Gilbert88'
```
Comment (vespian) [inline packages/dcos-integration-test/extra/test_mesos.py]: Adam is leaving Mesosphere, right?

Comment (orsenthil) [reply packages/dcos-integration-test/extra/test_mesos.py]: Since this is an open source repo, I thought, it is okay to include him?

Comment (vespian) [reply packages/dcos-integration-test/extra/test_mesos.py]: @adam-mesos What do you think? It would be great to have you still around :)

**Comment by orsenthil** [comment]
@vespian  - I have removed adam and included only active committers.

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by orsenthil** [comment]
@mesosphere-mergebot label ready for review

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Pull request merged**